src/map.cpp

```diff
@@ -3137,35 +3137,44 @@ void map::process_items( bool active, T processor, std::string signal )
 }
 
 template<typename T>
-void map::process_items_in_submap( submap *const current_submap, int gridx, int gridy, T processor,
-                                   std::string signal )
+void map::process_items_in_submap( submap *const current_submap, int const gridx, int const gridy,
+    T processor, std::string const &signal )
 {
     // Get a COPY of the active item list for this submap.
     // If more are added as a side effect of processing, they are ignored this turn.
     // If they are destroyed before processing, they don't get processed.
     std::list<item_reference> active_items = current_submap->active_items.get();
+    auto const p0 = point {gridx * SEEX, gridy * SEEY};
```

Comment:
Sure, got a suggestion for p0? Here I chose 0 as in `x = x0 + c` type expressions.